src/app/services/data/js-path.data.ts

```diff
@@ -3,6 +3,8 @@ import stage1 from '@/shared/assets/stages/stage-1.webp';
 import stage2 from '@/shared/assets/stages/stage-2.webp';
 import stage3 from '@/shared/assets/stages/stage-3.webp';
 
+// TODO Change 'topics' to string
```

Comment:
I agree with you, but in this case such a problem will not arise, since we have to refactor all `*.data` files and there is no point in creating another additional task for this changes

Reply:
Dont get it. If there is task to change it, then comment useless. If there is no task for this, it should be created and comment removed.

Reply:
We have a global task to refactor the `*.data` files. Within this global task, this file will also be refactored, respectively, the `array` will be changed to a `string` and this comment will be deleted. It makes no sense to create a separate task to replace the `array` with a `string` in this file, since it will simply be lost in the backlog and the issue will not be solved within the global refactoring `*.data` files.

Reply:
What I'm trying to say is that I don't get why a task or part of a task has to be right in the code.
If we have a specific issue for this task, - "refactor the *.data", then this task has to contain all steps and information for solving this task.

Whatever. I think this has to be discussed with everyone, because, as I see, not only your comments about "todo".